src/app/pages/license/license.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, NgModule } from "@angular/core";
```

Comment:
This file should be indented with 2 spaces instead of 4 and use single quotes instead of double for your import declarations.